package.json

```diff
@@ -51,28 +54,30 @@
     "sinon": "^4.3.0",
     "sinon-chai": "^3.2.0",
     "snappy": "^6.1.2",
-    "standard-version": "^4.4.0",
+    "standard-version": "^7.1.0",
+    "typescript": "^3.8.3",
     "worker-farm": "^1.5.0",
     "wtfnode": "^0.8.0",
     "yargs": "^14.2.0"
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=4"
+    "node": ">=10"
   },
   "bugs": {
     "url": "https://github.com/mongodb/node-mongodb-native/issues"
   },
   "scripts": {
-    "atlas": "node ./test/tools/atlas_connectivity_tests.js",
```

Comment:
Do we need to change script names here? 
This is one thing that seems deliberate / opinionated and not pragmatic.
Just calling it out.

Reply:
Yea it is totally opinionated and I'm motivated to go this way because I wanna help us stay organized with the addition of types into our tooling / publishing (and maybe more things soon, e.g. madge) I will double check that everyones on board

Reply:
There is a good embedded point here though: you need to make sure any evergreen tasks using these scripts are updated. For instance, the [atlas connectivity testing](https://github.com/mongodb/node-mongodb-native/blob/master/.evergreen/run-atlas-tests.sh#L10) scripts.

Reply:
Yep I made sure of that, unless you know of something that constructs npm run [script] commands I believe I found no other usages
https://github.com/mongodb/node-mongodb-native/pull/2277/files#diff-abe44f7ca927aad385d3db5ded372032L10-R10

Reply:
great!